Purge static files on delete (#2488)
opendatateam/udata: CHANGELOG.md

## Current (in progress)

- :warning: Deletion workflow changes [#2488](https://github.com/opendatateam/udata/pull/2488):
  - Deleting a resource now triggers the deletion of the corresponding static file
  - Deleting a dataset now triggers the deletion of the corresponding resources (including community resources) and their static files

>   - Adding a celery job to remove community resources not linked to a dataset

Reviewer: ```suggestion
  - Adding a celery job `purge-orphan-community-resources` to remove community resources not linked to a dataset. This should be scheduled regularly.
```

>   - ⚠️ Adding a migration file to populate resources fs_filename new field, and to delete orphaned resources files

Reviewer: ```suggestion
  - Adding a migration file to populate resources fs_filename new field. Scripts to delete the orphaned files are available here <link-to-a-gist>.
```

## Current (in progress)

- :warning: Deletion workflow changes [#2488](https://github.com/opendatateam/udata/pull/2488):
  - Deleting a resource now triggers the deletion of the corresponding static file

>   - Deleting a dataset now triggers the deletion of the corresponding community resources and their static files

Reviewer: add org and user too, plus a reference to the migration script and its impacts

(good CHANGELOG entry 👏)

>   - Deleting a dataset now triggers the deletion of the corresponding community resources and their static files

Reviewer: ```suggestion
  - Deleting a dataset now triggers the deletion of the corresponding resources (including community resources) and their static files
```
